nifi-toolkit/nifi-toolkit-encrypt-config/src/main/groovy/org/apache/nifi/properties/ConfigEncryptionTool.groovy

```diff
@@ -1467,18 +1470,18 @@ class ConfigEncryptionTool {
                 if (!tool.ignorePropertiesFiles || (tool.handlingFlowXml && existingNiFiPropertiesAreEncrypted)) {
                     // If we are handling the flow.xml.gz and nifi.properties is already encrypted, try getting the key from bootstrap.conf rather than the console
                     if (tool.ignorePropertiesFiles) {
-                        tool.keyHex = NiFiPropertiesLoader.extractKeyFromBootstrapFile(tool.bootstrapConfPath)
+                        tool.keyOrKeyId = NiFiPropertiesLoader.extractKeyFromBootstrapFile(tool.bootstrapConfPath)
                     } else {
-                        tool.keyHex = tool.getKey()
+                        tool.keyOrKeyId = tool.getKey()
                     }
 
-                    if (!tool.keyHex) {
+                    if (!tool.keyOrKeyId) {
                         tool.printUsageAndThrow("Hex key must be provided", ExitCode.INVALID_ARGS)
                     }
 
                     try {
                         // Validate the length and format
-                        tool.keyHex = parseKey(tool.keyHex)
+                        tool.keyOrKeyId = parseHexKey(tool.keyOrKeyId)
```

Comment:
@thenatog Hey, I think I'm missing something but this appears to block me from using `./bin/encrypt-config.sh` with a keystore provider since the key id isn't a hex key. To that point, I haven't been able to figure out exactly how to use the keystore provider correctly. What's the intended way of calling `encrypt-config.sh` when using the keystore provider?